jina/peapods/runtimes/asyncio/rest/models.py

```diff
@@ -51,6 +55,7 @@ class JinaRequestModel(BaseModel):
         Field(..., example=[Document().dict()])
     request_size: Optional[int] = default_request_size
     mime_type: Optional[str] = ''
+    # TODO mime type is contained in documents already
```

Comment:
So is this `mime_type` used? I can't see how

Reply:
I also don't see a reason to keep it there and I will create more tickets for further issues I discovered.

Reply:
Please note that, this follows the caller arguments from the [request_generator](https://github.com/jina-ai/jina/blob/master/jina/clients/request/__init__.py#L24)